.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "[go]": {
+    "editor.defaultFormatter": "golang.go"
+  },
+  "editor.formatOnPaste": true,
+  "go.formatTool": "goimports",
```

Comment:
This file will enforce the editor of the user who uses VSCode to use this setting